.circleci/config.yml

```diff
@@ -49,3 +49,132 @@ jobs:
+    parallelism: 1
+
+    working_directory: ~/drill
+
+    steps:
+    - checkout
+
+    - run:
+        name: Update packages list
+        command:
+          sudo apt-get update
+
+    - run:
+        name: Install java 9
+        command:
+          sudo apt-get -y install openjdk-9-jdk
+
+    - run:
+        name: Set default java 9
+        command:
+          sudo update-java-alternatives --set java-1.9.0-openjdk-amd64
+
+    - run:
+        name: Update maven version
+        command:
+          curl -fsSL https://git.io/vpDIf | bash -s -- 3.5.4
+
+    - run:
+        name: Install libaio1.so library for MySQL integration tests
+        command:
+          sudo apt-get install libaio1 libaio-dev
+    - run:
+        name: Drill project build
+        command:
+          mvn install -Drat.skip=false -Dlicense.skip=false -DmemoryMb=2560 -DdirectMemoryMb=4608 -DexcludedGroups="org.apache.drill.categories.SlowTest,org.apache.drill.categories.UnlikelyTest"
+
+  build_jdk10:
+    machine:
+      enabled: true
+      image: circleci/classic:latest
+    parallelism: 1
+
+    working_directory: ~/drill
+
+    steps:
+    - checkout
+
+    - run:
+        name: Update packages list
+        command:
+          sudo apt-get update
+
+    - run:
+        name: Install java 10
+        command:
+          sudo apt-get -y install openjdk-10-jdk
+
+    - run:
+        name: Set default java 10
+        command:
+          sudo update-java-alternatives --set java-1.10.0-openjdk-amd64
+
+    - run:
+        name: Update maven version
+        command:
+          curl -fsSL https://git.io/vpDIf | bash -s -- 3.5.4
+
+    - run:
+        name: Install libaio1.so library for MySQL integration tests
+        command:
+          sudo apt-get install libaio1 libaio-dev
+    - run:
+        name: Drill project build
+        command:
+          mvn install -Drat.skip=false -Dlicense.skip=false -DmemoryMb=2560 -DdirectMemoryMb=4608 -DexcludedGroups="org.apache.drill.categories.SlowTest,org.apache.drill.categories.UnlikelyTest"
+
+  build_jdk11:
+    machine:
+      enabled: true
+      image: circleci/classic:latest
+    parallelism: 1
+
+    working_directory: ~/drill
+
+    steps:
+    - checkout
+
+    - run:
+        name: Update packages list
+        command:
+          sudo apt-get update
+
+    - run:
+        name: Install java 11
+        command:
+          sudo apt-get -y install openjdk-11-jdk
+
+    - run:
+        name: Set default java 11
+        command:
+          sudo update-java-alternatives --set java-1.11.0-openjdk-amd64
+
+    - run:
+        name: Update maven version
+        command:
+          curl -fsSL https://git.io/vpDIf | bash -s -- 3.5.4
+
+    - run:
+        name: Install libaio1.so library for MySQL integration tests
+        command:
+          sudo apt-get install libaio1 libaio-dev
+    - run:
+        name: Drill project build
+        command:
+          mvn install -Drat.skip=false -Dlicense.skip=false -DskipTests
```

Comment:
Add the comment here why failures are obtained for tests on jdk11. 
Is it possible to disable one more tests group here?

Reply:
Great!